File: app/src/main/java/co/tpcreative/supersafe/model/BreakInAlertsModel.java
package co.tpcreative.supersafe.model;

import java.io.Serializable;

public class BreakInAlertsModel implements Serializable {
    public int id;
    public String fileName;
    public long time;
    public String pin;
    public BreakInAlertsModel(BreakInAlertsEntityModel index){
        this.id = index.id;
        this.fileName = index.fileName;
        this.time = index.time;
        this.pin = index.pin;
    }
    public BreakInAlertsModel(){
    }
}

File: app/src/main/java/co/tpcreative/supersafe/model/BreakInAlertsModel.kt
package co.tpcreative.supersafe.model
import java.io.Serializable

class BreakInAlertsModel : Serializable {
    var id = 0
    var fileName: String? = null
    var time: Long = 0
    var pin: String? = null

    constructor(index: BreakInAlertsEntityModel) {
        id = index.id
        fileName = index.fileName
        time = index.time
        pin = index.pin
    }

    constructor() {}
}